### DISCUSSION TITLE
[1545450]Incredibox

### DISCUSSION THREAD

**Discussion by nekoaweek** [pjy612/SteamManifestCache#624] [Share]
### Steam 游戏ID

1545450

### 密钥列表

```hash
1545451;59a399e17aace3e46fb1c7d7dead965a78c15b5b077b014e0b80247f51a08e5c
```

### 清单附件

[1545451.zip](https://github.com/pjy612/SteamManifestCache/files/15141746/1545451.zip)

### 备注
自从这一大堆白嫖方式（论坛发清单的，离线啦，跑路了的ikun，眼瞅出来blank密钥加密方式，我不知道他还收钱竟然），好像就没买过什么游戏了

既然要求不重复的游戏，还是翻我曾经阿根廷区买的游戏，应该还有好几个，但是这个分享格式要求也一次发一个。
这个Incredibox本来是手机游戏，玩法是混音音乐非常好听

**Comment by pjy612** [comment]
其实不算bug，freetoken，我故意放的。让外部能用用，毕竟初心还是清单下载器。
而且因为 BTM 那边 又公开了，Onekey 也不弄了，咱这个多一个不多 少一个不少。我也不想放使用说明。
反正 每小时5000访问，也就清单下载，除非他们有闲心去破解SAI...
至于拉人... 缘分到了自然...
至于盒... 我这id 感觉已经 近似实名上网了...

**Reply by nekoaweek** [reply]
简单说，就是这密钥权限贼tm大。能看所有组织人员，直接clone，登录github dekstop，然后还有QQ群都能看到（虽然但是这样直接说不会让别人香做什么坏事）

**Reply by nekoaweek** [reply]
至于blank，注意力极强的人可以直接：
通过观察，注意到，加密方式是如此这般

然后missingupdate，是我还要干啥

**Reply by pjy612** [reply]
有吗= =？貌似只是让他能查看和下载项目文件,当然 应该也支持 clone。但是 clone 后能不能用就是另外的事情了...
不过如果还能看 discussions 那就 有点意料之外了，愿闻其详

**Reply by pjy612** [reply]
什么还要干啥？
哦。。。意思是这个 只有你提供 自然没有后续更新了。。。标记一下罢了。
done 的 一般有更新。 miss update 那就是这独一份，不保证后续能更新的意思。
如果有人来这儿 借鉴密钥清单的话 根据 tag 能看出来。
至于缘分。。。 就是 继续保持。。。
